src/ray/protobuf/export_api/export_event.proto

```diff
@@ -18,11 +18,15 @@ option cc_enable_arenas = true;
 package ray.rpc;
 
 import "src/ray/protobuf/export_api/export_task_event.proto";
+import "src/ray/protobuf/export_api/export_actor_data.proto";
 
 // ExportEvent defines events stored by the export API. This
 // schema is public and any changes must be backward compatible.
 message ExportEvent {
-  enum SourceType { EXPORT_TASK = 0; }
+  enum SourceType {
+    EXPORT_TASK = 0;
+    EXPORT_ACTOR = 1;
+  }
```

Comment:
Actually we probably don't need SourceType since protobuf can let you know which oneof is set.

Reply:
Because the event is ultimately being written to file as json, the goal was for this proto schema to match the json schema. Let's leave this for now because this feature isn't released and public yet. When writing the docs if we add another way to share the json schema, we can update this then.